jsk_arc2017_common/config/label_names.yaml

```diff
@@ -0,0 +1,43 @@
+label_names:
```

Comment:
`list_objects.py` use objects.txt, but your `pick_json_generator.py` use label_names.yaml, so this is confusing.
I prefer data are placed in a one place to keep the consistency